templates/php/src/InputFile.php.twig

```diff
@@ -0,0 +1,49 @@
+<?php
+namespace {{ spec.title | caseUcfirst }};
+
+class InputFile {
+    private ?string $data;
+    private ?string $mimeType;
+    private ?string $filename;
+    private ?string $path;
+
+    public function __construct(){}
+
+    public function getData(): ?string
+    {
+        return $this->data;
+    }
+
+    public function getPath(): ?string
+    {
+        return $this->path;
+    }
+
+    public function getMimeType(): ?string
+    {
+        return $this->mimeType;
+    }
+
+    public function getFilename(): ?string
+    {
+        return $this->filename;
+    }
+
+    public static function withPath(string $path, ?string $mimeType = null, ?string $filename = null)
+    {
+        $instance = new InputFile();
+        $instance->path = $path;
+        $instance->mimeType = $mimeType;
+        $instance->filename = $filename;
+        return $instance;
+    } 
+
+    public static function withData(string $data, ?string $mimeType = null, ?string $filename = null)
```

Comment:
Can we come up with a better name? Data sounds too genetic. For example, in Node I called it `plaintext`, not sure if it's the same concept here.

Reply:
I've seen data been common for file data as string in PHP?

Reply:
Let's keep it if it's common in PHP.